### ISSUE TITLE
Including .py files causes them to appear in wheel twice

### ISSUE THREAD

**Issue by jwodder** [sdispater/poetry#482]
- [x] I am on the [latest](https://github.com/sdispater/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: Ubuntu Xenial 16.04.5
- **Poetry version**: 0.11.5
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: https://github.com/PeRDy/apistar-peewee-orm/blob/master/pyproject.toml (not mine, just an example)

## Issue
If the `include` field in `[tool.poetry]` matches any of the project's Python source files, then those files will be included in the wheel twice when building: running `zipinfo` on a wheel will produce output like:

```
Archive:  apistar_peewee_orm-0.3.5-py3-none-any.whl
Zip file size: 59146 bytes, number of entries: 17
-rw-r--r--  2.0 unx      250 b- stor 80-Jan-01 00:00 apistar_peewee_orm/__init__.py
-rw-r--r--  2.0 unx      250 b- stor 80-Jan-01 00:00 apistar_peewee_orm/__init__.py
-rw-r--r--  2.0 unx     3248 b- stor 80-Jan-01 00:00 apistar_peewee_orm/__main__.py
-rw-r--r--  2.0 unx     3248 b- stor 80-Jan-01 00:00 apistar_peewee_orm/__main__.py
-rw-r--r--  2.0 unx      745 b- stor 80-Jan-01 00:00 apistar_peewee_orm/components.py
-rw-r--r--  2.0 unx      745 b- stor 80-Jan-01 00:00 apistar_peewee_orm/components.py
-rw-r--r--  2.0 unx      233 b- stor 80-Jan-01 00:00 apistar_peewee_orm/database.py
-rw-r--r--  2.0 unx      233 b- stor 80-Jan-01 00:00 apistar_peewee_orm/database.py
-rw-r--r--  2.0 unx      934 b- stor 80-Jan-01 00:00 apistar_peewee_orm/event_hooks.py
-rw-r--r--  2.0 unx      934 b- stor 80-Jan-01 00:00 apistar_peewee_orm/event_hooks.py
-rw-r--r--  2.0 unx     4025 b- stor 80-Jan-01 00:00 apistar_peewee_orm/manager.py
-rw-r--r--  2.0 unx     4025 b- stor 80-Jan-01 00:00 apistar_peewee_orm/manager.py
?rw-------  2.0 unx       71 b- defN 16-Jan-01 00:00 apistar_peewee_orm-0.3.5.dist-info/entry_points.txt
-rw-r--r--  2.0 unx    35176 b- stor 80-Jan-01 00:00 apistar_peewee_orm-0.3.5.dist-info/LICENSE
?rw-------  2.0 unx       84 b- defN 16-Jan-01 00:00 apistar_peewee_orm-0.3.5.dist-info/WHEEL
?rw-------  2.0 unx     4987 b- defN 16-Jan-01 00:00 apistar_peewee_orm-0.3.5.dist-info/METADATA
?rw-------  2.0 unx     1489 b- defN 16-Jan-01 00:00 apistar_peewee_orm-0.3.5.dist-info/RECORD
17 files, 60677 bytes uncompressed, 56662 bytes compressed:  6.6%
```

and the `RECORD` file will contain two entries for each `.py` file.  Running `poetry build` will also produce warnings of the form:

```
/home/jwodder/tmp/poetry/venv/lib/python3.5/site-packages/poetry/masonry/builders/wheel.py:244: UserWarning: Duplicate name: 'apistar_peewee_orm/__init__.py'
  wheel.writestr(zinfo, src.read())
```

**Comment by sdispater** [comment]
This has been fixed in commit b546a1b.

It will land in the next `0.12` release.

**Comment by sdispater** [comment]
Version `0.12.0` has been released!
